test/extended/networking/ipsec.go

```diff
@@ -10,6 +10,7 @@ import (
 	"time"
 
 	v1 "github.com/openshift/api/operator/v1"
+	mg "github.com/openshift/origin/test/extended/machine_config"
```

Comment:
can you talk about the component that consumes the certs and what is this nss db? I want to understand how its consumed to ensure theres no race.